docs/_docs/02-ug-configuration.md

```diff
@@ -14,3 +14,5 @@ classes: wide
 | spark.hyperspace.explain.displayMode                 | "plaintext"                                                                                        | Display mode for Hyperspace explain() output. The valid set of values is: "console", "plaintext", "html".   | 0.1.0         |
 | spark.hyperspace.explain.displayMode.highlight.beginTag | "" (An empty string)                                                                                     | Tag to mark beginning of highlight portion in explain() output according to the display mode.         | 0.1.0         |
 | spark.hyperspace.explain.displayMode.highlight.endTag   | "" (An empty string)                                                                                    | Tag to mark ending of highlight portion in explain() output according to the display mode.            | 0.1.0         |
+| spark.hyperspace.index.lineage.enabled   | false                                                                                    | Add lineage column to index upon creation to track source data file for each index record. Lineage is required for an index to support delete in HybridScan and refresh in incremental or quick modes. Adding lineage will increase the size of the index, proportional to the number of distinct source data files the index is built on. | 0.3.0         |
```

Comment:
```suggestion
| spark.hyperspace.index.lineage.enabled   | false                                                                                    | Add lineage column to index upon creation to track source data file for each index record. Lineage is required to handle deleted files in HybridScan, or to refresh an index in the incremental mode. Adding lineage will increase the size of the index, proportional to the number of distinct source data files the index is built on. | 0.3.0         |
```

Reply:
I made this change and pushed @imback82  - But a quick question: Did you remove "quick" mode because it is not implemented yet? (Lineage is required for delete support in the quick mode as well).

Reply:
Yea, that's correct. We will update this when we have quick implemented. Thanks!